### PR TITLE
Rename pledge-sector

### PR DESCRIPTION
**Pull request by Kubuxu**

### CODE REVIEW

File: cmd/lotus-storage-miner/sectors.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 )
 
 var storeGarbageCmd = &cli.Command{
```
Comment (whyrusleeping) [inline cmd/lotus-storage-miner/sectors.go]: there are still a bunch of things called store garbage internally. We will want to change those too

Comment (Kubuxu) [reply cmd/lotus-storage-miner/sectors.go]: Done.

**Comment by jimmylee** [comment]
LGTM

File: documentation/en/mining.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ lotus-storage-miner info
 **Seal** random data to start producing **PoSts**:
 
 ```sh
-lotus-storage-miner store-garbage
+lotus-storage-miner pledge-sector
```
Comment (jimmylee) [inline documentation/en/mining.md]: LGTM

**Pull request merged**